recipes/ht/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "ht" %}
+{% set version = "0.1.52" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 75ba751e27aea1f6f025d7ee167096a46d888c410067d17d747f7d58d4a4ded1
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - fluids >=0.1.64
+    - numpy >=1.5.0
+    - python
+    - scipy >=0.9.0
+
+test:
+  imports:
+    - ht
+    - fluids
```

Comment:
I guess this isn't a test as fluids comes from a different package?
```suggestion
```
(assuming there isn't an optional circular dependencies that are being tested)

Reply:
Ah, yes. I put it here because fluids is an essential part. So I think its import could be asserted. But we can remove it, it has a test in its feedstock already. Thanks!